rai/qt/qt.cpp

```diff
@@ -243,10 +243,10 @@ void rai_qt::accounts::refresh_wallet_balance ()
 		balance = balance + (this->wallet.node.ledger.account_balance (transaction, key));
 		pending = pending + (this->wallet.node.ledger.account_pending (transaction, key));
 	}
-	auto final_text (std::string ("Wallet balance (XRB): ") + wallet.format_balance (balance));
+	auto final_text (std::string ("Balance: ") + wallet.format_balance (balance));
```

Comment:
This didn't require corresponding changes to the test case(s)?

Reply:
This is for viewing other accounts in the advanced menu, doesn't have a test AFAIK.